examples/05-nautobot-peeringdb/main.py

```diff
@@ -5,27 +5,35 @@
 from adapter_peeringdb import PeeringDB
 
 from diffsync.enum import DiffSyncFlags
+from diffsync.store.redis import RedisStore  # pylint: disable=no-name-in-module,import-error
 
+REDIS_HOST = "redis"
+PEERING_DB_IX_ID = 62  # CATNIX ID
+NAUTOBOT_URL = "https://demo.nautobot.com"
+NAUTOBOT_TOKEN = "a" * 40
```

Comment:
Bad code smell to me that we declare these constants separately in both `main.py` and `adapter_nautobot.py` - one or the other of these is redundant.

Reply:
I have removed from the adapter